app/components/SearchBox/SearchBox.tsx

```diff
@@ -0,0 +1,26 @@
+'use client';
+
+import React, { useState } from 'react';
+import { useRouter } from 'next/navigation';
+import styles from './SearchBox.module.scss';
+
+function SearchBox() {
+  const [input, setInput] = useState('');
+  const router = useRouter();
+
+  const handleSearch = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    router.push(`/search?q=${input}`);
+  };
+
+  return (
+    <form className={styles.searchBox} onSubmit={handleSearch}>
+      <input type="text" placeholder="Search keywords..." value={input} onChange={(e) => setInput(e.target.value)} />
+      <button type="submit" disabled={!input}>
```

Comment:
Please provide some styles for the disabled state, so it’ll be obvious from user experience that the button is disabled